### DISCUSSION TITLE
API springMVC

### DISCUSSION THREAD

**Discussion by liu729133700** [apache/skywalking#12519] [General]
I found a slow trace in the skywalking trace module. It spent almost 100% of its time on the API springMVC, I don't know what that api stands for, springMVC is just a framework for mapping URIs and Controllers, I don't know why it took so long, Is there a more convincing explanation that tells me what API springMVC stands for
<img width="1188" alt="slow_trace" src="https://github.com/user-attachments/assets/58fb6c37-995e-443e-b060-88a560658e4f">

**Comment by wu-sheng** [comment]
This URL is in your codes. Why do you ask us to tell?

**Reply by liu729133700** [reply]
I wonder if this springMVC represents the time between the request going into tomcat and it being processed, I can't find a sky walking description of API springMVC

**Reply by wu-sheng** [reply]
We would not describe that. Codes are there. The time is fron Tomcat codes if you have tomcat plugin.

**Reply by liu729133700** [reply]
ok, thank you！
